environment.yml

```diff
@@ -4,3 +4,8 @@ channels:
   - defaults
 dependencies:
   - gdal
```

Comment:
Is `gdal` used as a Python import anywhere? I did a quick pass-through and didn't see it.